test/index.js

```diff
@@ -4,25 +4,27 @@
  * Module dependencies.
  */
 
-const { CLIEngine } = require('eslint');
+const { ESLint } = require('eslint');
```

Comment:
The CLIEngine has been deprecated. However, it remains in v7 but it may be removed in a future major version.

https://eslint.org/docs/user-guide/migrating-to-7.0.0#the-cliengine-class-has-been-deprecated